Example/Chargebee/CBSDKProductsTableViewController.swift

```diff
@@ -72,7 +72,9 @@ extension CBSDKProductsTableViewController: ProductTableViewCellDelegate {
         func purchase(customerID: String) {
             self.view.activityStartAnimating(activityColor: UIColor.white, backgroundColor: UIColor.black.withAlphaComponent(0.5))
 
-            CBPurchase.shared.purchaseProduct(product: withProduct, customerId: customerID) { result in
+                //Create CBCustomerInfo Object and Pass as param which has (FirstName,LastName,Email).
+            let userInformation = CBCustomerInfo(first_name: "", last_name: "", email: "")
+            CBPurchase.shared.purchaseProduct(product: withProduct, customerId: customerID,userInfo: userInformation) { result in
```

Comment:
Can we update the tests wherever applicable?

Reply:
In Separate test cases ticket I will take-up and will add for now will proceed there is no proper setup right now to add tests.